.github/workflows/CI.yml

```diff
@@ -49,3 +49,19 @@ jobs:
       - uses: codecov/codecov-action@v1
         with:
           file: lcov.info
+          
+  slack:
+    name: Notify Slack Failure
+    needs: test
+    runs-on: ubuntu-latest
+    if: github.event == 'schedule'
+    steps:
+      - uses: technote-space/workflow-conclusion-action@v2
+      - uses: voxmedia/github-action-slack-notify-build@v1
+        if: env.WORKFLOW_CONCLUSION == 'failure'
+        with:
+          channel: nightly-rse
+          status: FAILED
+          color: danger
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.INVENIA_SLACK_BOT_TOKEN }}
```

Comment:
Is this secret defined in this project's secrets?

Reply:
Org secret.